### DISCUSSION TITLE
Set boundary conditions

### DISCUSSION THREAD

**Discussion by siwux** [neperfepx/FEPX#51] [Q&A] [answered]
I want to know how to set boundary conditions for pure shear simulation? @rquey

**Comment by mkasemer** [comment, marked as answer]
@siwux: one way to consider pure shear would be to utilize one of the triaxial conditions (either constant load rate or constant strain rate). The triaxial loading conditions apply loads normal to the surface, i.e., a macroscopic principal state. Recall that the principal state of `sig1 = A`, `sig2 = -A`, and `sig3 = 0` corresponds to pure shear loading .

You may also utilize the uniaxial loading module to apply loading in a direction orthogonal to the normal of the control surface, which is a simple shear condition.

**Reply by siwux** [reply]
Thank you very much for your reply.
